### DISCUSSION TITLE
Regarding issue #44 (bug: notify window gained focus unexpectedly)

### DISCUSSION THREAD

**Discussion by olatheander** [folke/snacks.nvim#125] [Q&A] [answered]
Since @folke closed issue #44 I don't want to spam by opening a similar issue I open a discussion thread instead.

I guess you can argue that it's not a bug in Notifier but after LazyVim replaced nvim-notify with Snack's notifier I got back the behavior that the notification window becomes a focus target.

To clarify; when moving between windows using `<C-w>` (or whatever you have mapped it to) e.g. to shift between the file and the NeoTree tree window and it happens to be a notification active the move will focus the notification thus requiring an extra command to actually put focus on the NeoTree. If there is no notification active you obviously focus NeoTree directly. Maybe not a bug per se but it is a bit annoying.

For nvim-notify this can be circumvented by having a block:
```
  {
    "rcarriga/nvim-notify",
    opts = {
      timeout = 2000,
      on_open = function(win)
        vim.api.nvim_win_set_config(win, { focusable = false }) -- Avoid focus on notification when navigating with <C-w> (https://github.com/rcarriga/nvim-notify/issues/182)
      end,
    },
  },
```
in the LV config and I believe that's what AbaoFromCUG requests.

**Comment by folke** [comment, marked as answer]
for snacks, set opts.styles.notification.focusable = false

**Reply by olatheander** [reply]
Excellent, thanks.
